spec_helper.rb

```diff
@@ -32,19 +32,11 @@ def init_publish_client(conf)
 end
 
 def init_dispatcher_client(conf)
-  Capybara.register_driver :chrome do |app|
-    Capybara::Selenium::Driver.new(app, browser: :chrome)
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, js_errors: false)
   end
 
-  Capybara.register_driver :headless_chrome do |app|
-    capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
-      chromeOptions: { args: %w[headless disable-gpu] }
-    )
-
-    Capybara::Selenium::Driver.new app, browser: :chrome, desired_capabilities: capabilities
-  end
-
-  Capybara.current_driver = :headless_chrome
+  Capybara.current_driver = :poltergeist
```

Comment:
Now that it's using Poltergeist driver on PhantomJS, please also update the README file to reflect the requirement to install PhantomJS (link to the poltergeist's PhantomJS installation guide) instead of ChromeDriver installation.